### PR TITLE
[skip ci] CODEOWNERS -= xiulipan

### PR DESCRIPTION
**Pull request by marc-hb**
Let's stop spamming Xiuli.

Signed-off-by: Marc Herbert <marc.herbert@intel.com>

### CODE REVIEW

**Review by fredoh9 — APPROVED** [review]

**Review by dbaluta — APPROVED** [review]

File: CODEOWNERS [hunk 1]
```diff
@@ -32,8 +32,7 @@ src/audio/drc/*				@johnylin76 @cujomalainey @dgreid
 src/audio/multiband_drc/*		@johnylin76 @cujomalainey @dgreid
 
 # platforms
-src/platform/baytrail/*			@xiulipan
```
Comment (cujomalainey) [inline CODEOWNERS:35]: lets find another owner for this

File: CODEOWNERS [hunk 2]
```diff
@@ -47,21 +46,20 @@ src/drivers/dw/*			@lyakh
...
-scripts/*				@xiulipan @marc-hb
+scripts/*				@marc-hb
 
 # tools(old 'soft' repo)
-tools/logger/*				@xiulipan @bkokoszx @akloniex
-tools/topology/*			@ranj063 @xiulipan
+tools/logger/*				@bkokoszx @akloniex
+tools/topology/*			@ranj063
 tools/testbench/*			@ranj063
-tools/test/*				@xiulipan
```
Comment (cujomalainey) [inline CODEOWNERS:64]: same here

**Review by XiaoyunWu6666 — COMMENTED** [review]
ok

**Review by lgirdwood — APPROVED** [review]

**Comment by lgirdwood** [comment]
@mengdonglin if you have no one then owner will be the maintainers (which is automatic).

**Pull request merged**